_events/2016-03-18-fossasia.md

```diff
@@ -10,7 +10,7 @@ status: past
 permalink: /2016/03/18/fossasia-2016
 ---
 
-[FOSSASIA](http://2016.fossasia.org/) is the premier Free and Open Source technology event in Asia for developers, start-ups, and contributors. Projects at FOSSASIA range from open hardware, to design, graphics and software. FOSSASIA was established in 2009. Previous events took place in Cambodia and Vietnam.
+[FOSSASIA](s) is the premier Free and Open Source technology event in Asia for developers, start-ups, and contributors. Projects at FOSSASIA range from open hardware, to design, graphics and software. FOSSASIA was established in 2009. Previous events took place in Cambodia and Vietnam.
```

Comment:
Is this correct? looks like a link was replaced with an 's' to me??

Reply:
@Arhell

Reply:
Done thanks